liblustreapi-types/build.rs

```diff
@@ -5,7 +5,7 @@
 use std::path::PathBuf;
 
 fn main() {
-    if cfg!(target_os = "macos") {
+    if cfg!(target_os = "macos") || cfg!(target_os = "windows") {
```

Comment:
May I suggest building on Linux only? Lustre is linux-only, no?

Reply:
I'll update.